sktime/transformations/series/cffilter.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+"""
+Interface to Christiano Fitzgerald asymmetric, random walk filter from `statsmodels`.
+
+Interfaces `cf_filter` from `statsmodels.tsa.filters`.
+"""
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["ken-maeda"]
+__all__ = ["CFFilter"]
+
+
+import pandas as pd
+
+from sktime.transformations.base import BaseTransformer
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+_check_soft_dependencies("statsmodels", severity="warning")
+
+
+class CFFilter(BaseTransformer):
+    """Filter a times series using the Christiano Fitzgerald filter.
+
+    This is a wrapper around the `cffilter` function from `statsmodels`.
+    (see `statsmodels.tsa.filters.cf_filter.cffilter`).
+
+    Parameters
+    ----------
+    low : float
+        Minimum period of oscillations. Features below low periodicity
+        are filtered out. Default is 6 for quarterly data,
+        giving a 1.5 year periodicity.
+
+    high : float
+        Maximum period of oscillations. Features above high periodicity
+        are filtered out. Default is 32 for quarterly data,
+        giving an 8 year periodicity.
+
+    drift : bool
+        Whether or not to remove a trend from the data.
```

Comment:
bit unclear what "remove" means. I assume it means "subtract"?
Also, `x` is not defined, you need to say this refers to a univariate `X` seen in `transform`.

Reply:
The sentense "remove trend" somethime is used in time series analysis, but it completely is same meaning to substracting trend from the sequence. So I'll replace it.

Reply:
I know, but it does not always have the same meaning. You could also de-trend multiplicatively. It's worth being as precise as possible, since people from a different user context may use the class.